pkg/app/piped/eventwatcher/eventwatcher.go

```diff
@@ -223,28 +227,60 @@ func (w *watcher) updateValues(ctx context.Context, repo git.Repo, events []conf
 	}
 	defer tmpRepo.Clean()
 
-	handledEventIDs := make([]string, 0, len(events))
+	eventResults := make([]*pipedservice.ReportEventStatusesRequest_Event, 0, len(events))
 	for _, e := range events {
 		latestEvent, ok := w.eventGetter.GetLatest(ctx, e.Name, e.Labels)
 		if !ok {
 			continue
 		}
-		if latestEvent.Handled {
+		if latestEvent.IsHandled() {
 			continue
 		}
 		if err := w.commitFiles(ctx, latestEvent.Data, e, tmpRepo, commitMsg); err != nil {
 			w.logger.Error("failed to commit outdated files", zap.Error(err))
+			eventResults = append(eventResults, &pipedservice.ReportEventStatusesRequest_Event{
+				Id:                latestEvent.Id,
+				Status:            model.EventStatus_EVENT_FAILURE,
+				StatusDescription: fmt.Sprintf("Failed to change files: %v", err),
+			})
 			continue
 		}
-		handledEventIDs = append(handledEventIDs, latestEvent.Id)
+		eventResults = append(eventResults, &pipedservice.ReportEventStatusesRequest_Event{
+			Id:                latestEvent.Id,
+			Status:            model.EventStatus_EVENT_SUCCESS,
+			StatusDescription: fmt.Sprintf("Successfully handled by %q piped", w.config.PipedID),
```

Comment:
UUIDs like PipeID are hard to read for humans. So I think we can eliminate it from the description string.
The description strings could be something like "Successfully updated %d files in repository %s".

Reply:
For sure, Piped doesn't matter that much.

Reply:
Updated